main/modes/mode_tunernome.c

```diff
@@ -823,18 +826,14 @@ void tunernomeMainLoop(int64_t elapsedUs)
                     // Plot text on top of everything else
                     uint8_t semitoneNum = (tunernome->curTunerMode - SEMITONE_0);
                     bool shouldDrawFlat = (semitoneNoteNames[semitoneNum][strlen(semitoneNoteNames[semitoneNum]) - 1] == 1);
-                    int16_t tWidth = textWidth(&tunernome->mm, semitoneNoteNames[semitoneNum]);
+                    char buf[5] = {0};
+                    strncpy(buf, semitoneNoteNames[semitoneNum], 4);
```

Comment:
Here too, I don't quite get the reason for copy.

Reply:
See first answer